code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -38,6 +38,7 @@
 	message_admins("[key_name_admin(imp_in)]<A HREF='?_src_=holder;adminmoreinfo=\ref[imp_in]'>?</A> (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[imp_in]'>FLW</A>) has activated their [name] at <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[imp_in.x];Y=[imp_in.y];Z=[imp_in.z]'>[A.name] (JMP)</a>.")
 //If the delay is short, just blow up already jeez
 	if(delay <= 7)
+		explosion(src,0,1,0,0,flame_range = 0)
 		explosion(src,heavy,medium,weak,weak, flame_range = weak)
```

Comment:
Isn't the fifth argument flash_range?? It probably shouldn't be the same as weak